CONTRIBUTING.md

```diff
@@ -574,13 +570,10 @@ include("feature2.jl")
 end
 ```
 
-* Files that declare modules should have the same name name of the module.
+- Files that declare modules should have the same name name of the module.
   E.g the module `SomeModule` is declared under the file `SomeModule.jl`.
 
-* Separate logical blocks of code with one blank line,
```

Comment:
I didn't mean that this should totally disappear, in particular, the separation of logical blocks of code with one blank line is a good guideline.